src/server.jl

```diff
@@ -48,7 +48,7 @@ function init!(file::File)
 end
 
 function refresh!(file::File, frontmatter::Dict)
-    exeflags = frontmatter["julia"]["exeflags"]
+    exeflags = frontmatter["format"]["metadata"]["julia"]["exeflags"]
```

Comment:
this should probably not be called `frontmatter` anymore then

Reply:
Indeed